### PR TITLE
Load user data

### PR DESCRIPTION
**Pull request by Hwesta**
* Stub out user_data models
* Add test loading of user data into models
* Add a first-pass achievements implementation
* Add a couple sample achievements
* Add bird species fixture
* Dev dependency for fancy manage.py commands
* Update README with install instructions & problems

### CODE REVIEW

File: user_data/models.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
+from django.db import models
+from django.conf import settings
+from django.contrib.gis.db import models as gismodels
```
Comment (dfloer) [inline user_data/models.py]: django.contrib.gis.db imports everything in django.db.models

File: user_data/views.py [hunk 2]
```diff
@@ -1,3 +1,91 @@
...
+            id=int(entry['Submission ID'][1:]),  # Strip leading S
+            user=user,
+            location=location,
+            defaults={
+                'complete_checklist': True if entry['All Obs Reported'] == '1' else False,
+                'start_date_time': start,
+                'checklist_comments': entry['Checklist Comments'] or '',
+                'number_of_observers': entry['Number of Observers'],
+                'protocol': entry['Protocol'],
+                'duration': entry['Duration (Min)'] or '0',
```
Comment (dfloer) [inline user_data/views.py]: Convert to DurationField.

File: user_data/views.py [hunk 3]
```diff
@@ -1,3 +1,91 @@
...
+                'distance': entry['Distance Traveled (km)'] or '0',
```
Comment (dfloer) [inline user_data/views.py]: Convert to DecimalField.

**Review by dfloer — COMMENTED** [review]
Find out what license the Clement's checklist is under and if it's safe to include as user_data/fixtures/ebird-taxonomy-aug2016.json in the project, or if we need to have whoever is using the code supply it and have our code parse and load it.

File: user_data/models.py (hunk 1 above)
Comment (Hwesta) [reply user_data/models.py]: Done

File: user_data/views.py (hunk 2 above)
Comment (Hwesta) [reply user_data/views.py]: Done

File: user_data/views.py (hunk 3 above)
Comment (Hwesta) [reply user_data/views.py]: Done

**Pull request merged**